processing/src/main/java/io/druid/query/Druids.java

```diff
@@ -104,9 +120,9 @@ public static AndDimFilterBuilder newAndDimFilterBuilder()
 
   /**
    * A Builder for OrDimFilter.
-   * 
+   * <p/>
```

Comment:
can we change the default template to not add those empty tags, Java 8 javadocs does not like them